app/views/user_mailer/_request_expired.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Your <%= link_to 'reservation request', reservation_url(@reservation, only_path: false) %> has expired.</p>
+
+<p> You can request the equipment again <%= link_to 'here', equipment_model_url(@reservation.equipment_model) %>.</p>
```

Comment:
Whoops, another comment - shouldn't the `equipment_model_url` call have `only_path: false` passed as an option since this is an e-mail?